pkg/manager/backup/cassandra/handler/grpc/handler.go

```diff
@@ -52,25 +48,26 @@ func New(opts ...Option) Server {
 }
 
 func (s *server) GetVector(ctx context.Context, req *payload.Backup_GetVector_Request) (res *payload.Backup_Compressed_MetaVector, err error) {
-	meta, err := s.cassandra.GetMeta(ctx, req.Uuid)
+	uuid := req.GetUuid()
+	meta, err := s.cassandra.GetMeta(ctx, uuid)
 	if err != nil {
-		detail := errDetail{method: "GetVector", uuid: req.Uuid}
 		if errors.IsErrCassandraNotFound(errors.UnWrapAll(err)) {
 			log.Warnf("[GetVector]\tnot found\t%v\t%+v", req.Uuid, err)
-			return nil, status.WrapWithNotFound("cassandra entry not found", &detail, err)
+			return nil, status.WrapWithNotFound(fmt.Sprintf("GetVector API cassandra uuid %s's object not found", uuid), err, info.Get())
 		}
 		log.Errorf("[GetVector]\tunknown error\t%+v", err)
-		return nil, status.WrapWithUnknown("Unknown error occurred", &detail, err)
+		return nil, status.WrapWithUnknown(fmt.Sprintf("GetVector API cassandra uuid %s's unknown error occurred", uuid), err, info.Get())
 	}
 
 	return toBackupMetaVector(meta)
 }
 
 func (s *server) Locations(ctx context.Context, req *payload.Backup_Locations_Request) (res *payload.Info_IPs, err error) {
-	ips, err := s.cassandra.GetIPs(ctx, req.Uuid)
+	uuid := req.GetUuid()
+	ips, err := s.cassandra.GetIPs(ctx, uuid)
 	if err != nil {
 		log.Errorf("[Locations]\tunknown error\t%+v", err)
```

Comment:
![badge](https://img.shields.io/badge/review-must-red.svg?style=flat-square)
Please revise this error message to be "not found error".

Reply:
may be there is alot of places that I have to fix, but I don't want to do that because all of previous handling is unknown I just rebase each places, may be we have to fix it later or another PR's. or after debug we should delete this line.
I think if error in gRPC handler we should execute logging in internal/net/grpc/status.newStatus

Reply:
Okay.

Reply:
but totally I agree with your opinion